samples/main/main.go

```diff
@@ -31,43 +33,46 @@ func main() {
 	// add console logger (optional)
 	logging.NewConsoleLogger(logging.InfoLevel, logging.SeparateFormat, true)
 	// add file logger (optional)
-	//logging.NewFileLogger("/Users/J/logs/opensergo/opensergo-universal-transport-service.log", logging.InfoLevel, logging.JsonFormat, true)
-
-	// instant OpenSergoClient
-	openSergoClient := client.NewOpenSergoClient("127.0.0.1", 10246)
+	// logging.NewFileLogger("./opensergo-universal-transport-service.log", logging.InfoLevel, logging.JsonFormat, true)
 
-	// registry SubscribeInfo of FaultToleranceRule
-	// 1. instant SubscribeKey
-	faultToleranceSubscribeKey := subscribe.NewSubscribeKey("default", "foo-app", configkind.ConfigKindRefFaultToleranceRule{})
-	// 2. instant Subscriber
-	sampleFaultToleranceRuleSubscriber := new(samples.SampleFaultToleranceRuleSubscriber)
-	// 3. construct SubscribeInfo
-	faultToleranceSubscribeInfo := client.NewSubscribeInfo(faultToleranceSubscribeKey)
-	faultToleranceSubscribeInfo.AppendSubscriber(sampleFaultToleranceRuleSubscriber)
-	// 4. registry
-	openSergoClient.RegisterSubscribeInfo(faultToleranceSubscribeInfo)
+	// Create a OpenSergoClient.
+	openSergoClient, err := client.NewOpenSergoClient("127.0.0.1", 10246)
+	if err != nil {
+		log.Fatalln("Failed to create OpenSergoClient: " + err.Error())
```

Comment:
`log.Fataln()` will invoke `os.Exit(1)`, the hole application will be shutdown. May we wait for keeping alive ? I think  only log the messages and left a `// TODO` to handle by user self. Just like follwing:
``` go
logging.Error("Failed to create OpenSergoClient: " + err.Error())
// TODO the error logic
```

Reply:
Actually some of the errors are not recoverable. The handling logic depends on developers (thinking it fatal or not). 

When `Start` the client, it's indeed weird to return an error  but has a keep-alive loop internal (and the client may finally recover...). We may need to improve the auto-keep-alive model and error model of the client to make it more clear. We may discuss this in a new issue.

Reply:
I've simplified the sample, but it needs to be updated again once the auto-keep-alive model and error model has been settled down.